src/wrapper.js

```diff
@@ -4,26 +4,18 @@ import Autocomplete from './autocomplete'
 function accessibleAutocomplete (opts) {
   if (!opts.element) { throw new Error('element is not defined') }
   if (!opts.id) { throw new Error('id is not defined') }
-  if (!opts.source) { throw new Error('source is not defined') }
+  if (!opts.source && !opts.values) { throw new Error('values or source are not defined') }
```

Comment:
Also, come to think of it, update the [usage examples](https://github.com/alphagov/accessible-autocomplete#using-npm-and-a-module-system) to use this new syntax, because it's much friendlier.